### PR TITLE
Update README.md

### LINKED ISSUE #117: Update README.md to describe how we put together deb and rpms
The main READMe file does not link to the linux subdir readme file to describe how those installers are built.

### PR DESCRIPTION
**Pull request by 2001asjad**
Added steps to describe how we put together deb and rpms in the README file.

Fixes #117

### CODE REVIEW

**Comment by karianna** [comment]
@2001asjad You'll need to sign the Eclipse Contributor Agreement (Follow your nose from https://www.eclipse.org/legal/ECA.php).

**Review by karianna — CHANGES_REQUESTED** [review]
Change this to a simple link to the README.md file under the linux sub folder :0-)

**Comment by 2001asjad** [comment]
> Change this to a simple link to the README.md file under the linux sub folder :0-)

Sorry I am a bit confused. Can you please explain me a bit more about it. Thanks

File: README.md [hunk 1]
```diff
@@ -4,8 +4,15 @@ Repository for creating installable packages for AdoptOpenJDK releases.
 The packages are created using:
 1. The Wix Toolset http://wixtoolset.org (Windows only)
 2. [Packages](http://s.sudre.free.fr/Software/Packages/about.html) (Mac OS)
-3. ??? (Linux .deb)
-4. ??? (Linux .rpm)
+3. To install `.deb` file via Command Line, follow these steps:
```
Comment (karianna) [inline README.md:7]: You can replace all of this with something like `3. [linux/README.md](Linux)`

**Comment by 2001asjad** [comment]
@karianna addressed the changes, please take a look.
Thanks

**Review by karianna — APPROVED** [review]
lgtm

**Pull request merged**